bp_me/test/tb/bp_cce_uc/bsg_trace_rom.py

```diff
@@ -3,8 +3,9 @@
 from trace_gen import TraceGen
 from npa_addr_gen import NPAAddrGen
 
-
-tg = TraceGen(addr_width_p=39, data_width_p=64)
+paddr_width = 40
+uc_mask = (1 << (paddr_width-1))
```

Comment:
This no longer determines cached/uncached in our cache.  Now it's < 0x8000_0000 for uncached.  I assume your mock LCE uses this old system so it's fine, just a heads up

Reply:
Correct, the mock LCE still uses the msb for cached/uncached, and these tests set the DRAM offset to 0 in bp_mem because the CCE connects directly to bp_mem. I will add a comment in this file and the mock LCE.